source/javascripts/controllers/_YMLController.js.erb

```diff
@@ -10,7 +10,7 @@ import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
 			var editor;
 
 			viewModel.appSlug = appService.appDetails.slug;
-			viewModel.shouldShowYmlStorageSettings = false;
+			viewModel.shouldShowYmlStorageSettings = requestService.isWebsiteMode();
```

Comment:
I think at this point this should be false, and only set it to (potentially) true after the asynchronous loading of the pipeline config.

Otherwise, if a user visits this tab on the website & the pipeline config is not yet loaded, this flag will be `true` for a short period, which makes `r-yml-storage-settings` appear with not yet initialized properties.